src/getlikes.js

```diff
@@ -0,0 +1,12 @@
+// const getLikes = () => fetch('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/8WhiXHgGMaGrsfo6vYsR/likes')
+//   .then((response) => response.json())
+//   .then((data) => data);
+
+// export default getLikes;
```

Comment:
commented code makes your file look not so good, I suggest removing it for better code organizatio